.github/workflows/test.yml

```diff
@@ -136,3 +138,43 @@ jobs:
           cd test-projects/rebar3
           rebar3 ct
         if: ${{matrix.combo.rebar3-version}}
+
+  integration_test_windows:
+    name: >
+      Pre-release integration tests
+      (Windows ${{matrix.combo.os}},
+       Erlang/OTP ${{matrix.combo.otp-version}},
+       rebar3 ${{matrix.combo.rebar3-version}})
```

Comment:
This looks better. Let me just get @starbelly's input on the previous question then I'll handle them at the same time.

> I'm not sure why rebar3-version didn't show up in the screenshot above.

Me neither: checking... well, in that case we didn't choose a `rebar3` on purpose, and I couldn't figure out how to have conditional titles, so that's the best I could do. We could just remove that info and put it in the step name, for example (not tested yet).